massgov/mayflower-patternlab
# Dp 7786 create sort by variant

Create the sort by component as designed in the search requirements document sort section in Mayflower react.

## Description

create new button toggle atom to toggle between two values

## Related Issue / Ticket

- [JIRA issue DP-7786](https://jira.state.ma.us/browse/DP-7786)
- [Github issue]()

## Steps to Test


1. go to http://localhost:3000/?p=atoms-button-toggle
2. verify the pattern look like the design in the requirement doc: 
![screen shot 2018-02-27 at 11 47 58 am](https://user-images.githubusercontent.com/5789411/36742323-60e82e0a-1bb5-11e8-9092-856224056c1a.png)

## Additional Notes:
Use this pattern in resultHeading molecule: https://mayflower.digital.mass.gov/?p=molecules-results-heading&view=c

#### Impacted Areas in Application

* 

#### @TODO

import this pattern into mayflower-react and use this pattern in resultHeading molecule for search
